config/default_config.yml

```diff
@@ -15,6 +15,7 @@ server:
 services:
   geocoder:
     url: override_by_environment
+    useLang: true
```

Comment:
is it the default value we want ?

Reply:
I think so. It has no impact for now, and will be helpful as soon as the geocoder is released.